.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: monthly
+    reviewers:
+      - willkg
+  - package-ecosystem: docker
+    directory: /docker
+    schedule:
+      interval: monthly
+    reviewers:
+      - willkg
```

Comment:
I cribbed some of this from gcp-ingestion. They also use labels, but socorro doesn't use lables except for the "DO NOT MERGE" one.